Roles/Crewmate/Mole.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using static TOHE.Options;
+
+namespace TOHE.Roles.Crewmate
+{
+    public static class Mole
+    {
+        private static readonly int Id = 26000;
+        private static List<byte> playerIdList = new();
```

Comment:
I think that `playerIdList` can be removed, since it is not used in the code

Reply:
> I think that `playerIdList` can be removed, since it is not used in the code

done